src/errorpage.cc

```diff
@@ -179,21 +178,23 @@ error_hard_text[] = {
     {
         ERR_REQUEST_START_TIMEOUT,
         "request start timedout"
-    },
+    }
+}};
+
+/// \ingroup ErrorPageInternal
```

Comment:
We do not use these Doxygen grouping anymore.

```suggestion
```

Reply:
Done.